web/server.go

```diff
@@ -3,40 +3,34 @@ package web
 import (
 	"context"
 	"net/http"
-	"time"
 
-	sentrygin "github.com/getsentry/sentry-go/gin"
 	"github.com/gin-gonic/gin"
-	"github.com/sirupsen/logrus"
 	"go.mongodb.org/mongo-driver/mongo"
 
+	"github.com/bitmark-inc/bitmark-sdk-go/account"
 	"github.com/bitmark-inc/data-store/store"
 )
 
-var log *logrus.Entry
-
-func init() {
-	log = logrus.WithField("prefix", "gin")
-}
-
 // Server to run a http server instance
 type Server struct {
 	server *http.Server
 
 	dataStorePool store.DataStorePool
 
+	bitmarkAccount *account.AccountV2
+
 	location string
 	rootKey  []byte
 }
 
 // NewServer new instance of server
-func NewServer(mongoClient *mongo.Client) *Server {
-
+func NewServer(location string, mongoClient *mongo.Client, acct *account.AccountV2) *Server {
 	return &Server{
-		location: "localhost",
+		location: location,
 		rootKey:  []byte("hello world"),
```

Comment:
Set the root key using another variable and get from the environment. For example, `viper.GetString("server.macaroon_root_key")`